common/src/main/java/org/valkyrienskies/mod/mixin/feature/render_pathfinding/MixinDebugRenderer.java

```diff
@@ -0,0 +1,31 @@
+package org.valkyrienskies.mod.mixin.feature.render_pathfinding;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.client.renderer.MultiBufferSource.BufferSource;
+import net.minecraft.client.renderer.debug.DebugRenderer;
+import net.minecraft.client.renderer.debug.PathfindingRenderer;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(DebugRenderer.class)
+public class MixinDebugRenderer {
+
+    @Shadow
+    @Final
+    public PathfindingRenderer pathfindingRenderer;
+
+    @Inject(method = "render", at = @At("HEAD"))
+    void render(
+        final PoseStack matrixStack,
+        final BufferSource buffer,
+        final double camX, final double camY, final double camZ,
+        final CallbackInfo ci) {
+
+        pathfindingRenderer.render(matrixStack, buffer, camX, camY, camZ);
```

Comment:
If we're leaving this in, can you add a normal config option to enable/disable it (and leave a comment on that config option that the variable in the mixin config plugin needs to be set)

Reply:
Maybe under some 'advanced' config subcategory